compat_tester/webauthn-rs-demo/Cargo.toml

```diff
@@ -11,21 +11,27 @@ repository = "https://github.com/kanidm/webauthn-rs"
 readme = "README.md"
 license = "MPL-2.0"
 
+[features]
+default = ["tls"]
+tls = ["dep:tide-openssl"]
```

Comment:
We always need openssl anyway, is there a reason to feature gate this?

Reply:
It was mainly gated because of #357; but if we go straight to this then we don't need it at all.

Reply:
I think we just go to openssl here and don't bother with the feature gates.